orga/tests/integration/components/routes/authenticated/campaign/profile/list-test.js

```diff
@@ -150,4 +152,121 @@ module('Integration | Component | routes/authenticated/campaign/profile/list', f
       assert.contains('En attente de profiles');
     });
   });
+
+  module('when user works for a SCO organization which manages students', function() {
+    class CurrentUserStub extends Service {
+      prescriber = { areNewYearSchoolingRegistrationsImported: false }
```

Comment:
bizarre que le linter dise rien ici :
```suggestion
      prescriber = { areNewYearSchoolingRegistrationsImported: false },
```